web/components/dropdowns/cycle.tsx

```diff
@@ -215,39 +250,57 @@ export const CycleDropdown: React.FC<Props> = observer((props) => {
                 cycle={selectedCycle}
                 className={buttonClassName}
                 dropdownArrow={dropdownArrow && !disabled}
+                dropdownArrowClassName={dropdownArrowClassName}
+                hideIcon={hideIcon}
+                placeholder={placeholder}
               />
             ) : buttonVariant === "border-without-text" ? (
```

Comment:
Please make sure to change this logic later on, on how these can be rendered and separate out the different buttons to a common file rather than redefining them in each of the dropdowns